### DISCUSSION TITLE
Does anyone have an example of for client c+?

### DISCUSSION THREAD

**Discussion by ronculyer** [valkey-io/valkey#716] [Q&A]
Loving this new tool but i am curious if anyone has an example of how to use this on client side on c++? im struggling to find how to connect to the server remotely from c++ as setting and getting.

**Comment by ranshid** [comment]
I think you have many options but the fastest that comes into my mind is to use [libvalkey](https://github.com/valkey-io/libvalkey) which is a c client for valkey (much like hiredis) which you can link to your C++ application.

**Comment by dmazzella** [comment]
take a look at the [Boost.Redis](https://www.boost.org/doc/libs/1_85_0/libs/redis/doc/html/index.html) library.
